### PR TITLE
Tool for uploading to HF Hub

### PR DESCRIPTION
**Pull request by farzadab**
The tool can be used as follows to 1) load a W&B model artifact 2) merge the LoRA weights and 3) upload them to HuggingFace Hub as follows:
```bash
python -m ultravox.tools.push_to_hub -m wandb://<model_path> --hf_upload_url fixie-ai/ultravox
```

### CODE REVIEW

File: ultravox/tools/push_to_hub.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+@dataclasses.dataclass
+class UploadToHubArgs:
+    # Model ID to use for the model
+    model: str = simple_parsing.field(alias="-m")
+    # Audio processor ID to use
+    audio_processor: Optional[str] = None
+    # Tokenizer ID to use
+    tokenizer: Optional[str] = None
+    # Data sets to use for inference
```
Comment (juberti) [inline ultravox/tools/push_to_hub.py:19]: This seems off

File: ultravox/tools/push_to_hub.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    device: Optional[str] = simple_parsing.field(default=None, alias="-D")
+    # Data type to use for the model
+    data_type: Optional[str] = None
+    # HuggingFace Hub URL to push to
+    hf_upload_url: Optional[str] = None
```
Comment (juberti) [inline ultravox/tools/push_to_hub.py:24]: this isn't a URL, more like a `hf_upload_model`?

File: ultravox/tools/push_to_hub.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+import dataclasses
+from typing import Optional
+
+import simple_parsing
+
+from ultravox.inference import ultravox_infer
+
+
```
Comment (juberti) [inline ultravox/tools/push_to_hub.py:10]: Suggest adding a comment with an example command line, as in the PR description

**Review by juberti — APPROVED** [review]
Nice!

File: ultravox/tools/push_to_hub.py (hunk 3 above)
Comment (farzadab) [reply ultravox/tools/push_to_hub.py:10]: All done.

File: ultravox/tools/push_to_hub.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+# This script is used to upload a model to the HuggingFace Hub. It is mainly useful for turning a W&B artifact into a HF model for vLLM use.
```
Comment (juberti) [inline ultravox/tools/push_to_hub.py:11]: more likely, for public use

**Review by juberti — APPROVED** [review]

**Pull request merged**